src/components/Admin/AdminActive.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <v-container
+    fluid
+  >
+    <v-card
+      class="mb-16 grey lighten-4"
+      style="height: 100%;"
+    >
+      <v-system-bar />
+      <v-toolbar
+        flat
+        height="65%"
+      >
+        <v-toolbar-side-icon
+          :left="true"
+        >
+          <img
+            class="mr-3"
+            src="https://brand.uark.edu/_resources/images/shield.jpg"
+            height="55px"
+            width="55px"
+          >
+        </v-toolbar-side-icon>
+        <v-toolbar-title
+          class="text-h5"
+        >
+          Welcome,
+          <a
+            href="/#/Profile"
+            class="text-decoration-none red--text text--darken-4"
+          >
+            Admin
+          </a>
+        </v-toolbar-title>
+        <v-btn
+          depressed
+          :right="true"
```

Comment:
:right="true" is the same thing as just writing right. Change them both to `right` and `absolute` to make it more readable.
```suggestion
          right
          absolute
```